### ISSUE TITLE
Code command causes window to fade out

### ISSUE THREAD

**Issue by wt-l00** [microsoft/vscode#136330]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.61.2
- OS Version: MacOS Monterey 12.0.1 

Steps to Reproduce:

1. `code sample.c --verbose`

The output is
```
| 12:33:44 | ❯ code sample.c --verbose  
[main 2021-11-03T03:33:53.665Z] Sending env to running instance...
[main 2021-11-03T03:33:53.696Z] Sent env to running instance. Terminating...
[main 2021-11-03T03:33:53.697Z] Lifecycle#kill()
```

**Comment by wt-l00** [comment]
may be related with #134635

**Comment by deepak1556** [comment]
It looks like there is already a running instance of code, please try to close all code related processes and try the command again. If the problem persists please share a screen capture of what is not working.

/gifPlease

**Comment by wt-l00** [comment]
@deepak1556 
I'm sorry, my description of this problem was not accurate.
Thanks for your comment.

The problem is that when I type a code command, it fades out of the window that is currently open in vscode.
I can open the file I want to open with the code command. However, I don't know why it fades out from the current window.

Here is a capture of the screen.


https://user-images.githubusercontent.com/29685855/140436304-50d07c07-59ba-40f6-9628-37de97271315.mov

**Comment by deepak1556** [comment]
Thanks for the additional details it clarifies the issue, what you are facing is https://github.com/microsoft/vscode/issues/134635. This should be addressed with `1.63` insiders.

/duplicate https://github.com/microsoft/vscode/issues/134635
